### ISSUE TITLE
How about onDragEnter handler?

### ISSUE THREAD

**Issue by denysdovhan** [feross/drag-drop#26]
Thanks for package!

It would be great if we could do something like this:

``` js
dragDrop('.container', {
  onDrop(files, pos) {
    console.log('Here are the dropped files', files)
    console.log('Dropped at coordinates', pos.x, pos.y)
  },

  // ---

  onDragEnter() {
    // ...
  }

  // ---

  onDragLeave(e) {
    console.log('dragend', e);
  }
});
```

What do you think?

**Comment by feross** [comment]
Sure, this is easy to support. Released as 2.13.0.

**Comment by denysdovhan** [comment]
@feross thanks! Nice job!
